.github/workflows/create-release.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish to Chrome Web Store.
+name: Create a release to publish
```

Comment:
Please revert all the changes in this file. The name should be what it is. The indentation is correct at the moment.